test/bigtest/tests/units/settings/MappingProfileDetails/MappingProfileDetails-test.js

```diff
@@ -0,0 +1,210 @@
+      expect(mappingProfileDetails.summary.description.value.text).to.equal('AP Holdings 1 description');
+      expect(mappingProfileDetails.summary.recordType.value.text).to.equal(commonTranslations['recordTypes.holdings']);
+      expect(mappingProfileDetails.summary.outputFormat.value.text).to.equal('MARC');
+    });
+
+    it('should display add transformation button', () => {
+      expect(mappingProfileDetails.transformations.addButton.isPresent).to.be.true;
+    });
+
+    it('should display add transformation button', () => {
+      expect(mappingProfileDetails.transformations.addButton.isPresent).to.be.true;
+    });
+
+    it('should display correct transformations fields headers', () => {
+      expect(mappingProfileDetails.transformations.list.headers(0).text).to.equal(translations['mappingProfiles.transformations.fieldName']);
+      expect(mappingProfileDetails.transformations.list.headers(1).text).to.equal(translations['mappingProfiles.transformations.transformation']);
+    });
+
+    it('should display correct transformations values', () => {
+      expect(mappingProfileDetails.transformations.list.rows(0).cells(0).text).to.equal('Holdings - Call number');
+      expect(mappingProfileDetails.transformations.list.rows(0).cells(1).text).to.equal('test');
+    });
+  });
+
+  describe('rendering mapping profile details', function () {
+    const handleSubmitSpy = sinon.spy();
+
+    beforeEach(async function () {
+      handleSubmitSpy.resetHistory();
+
+      await mountWithContext(
+        <Paneset>
+          <Router>
+            <MappingProfileDetails
+              mappingProfile={{
+                hasLoaded: true,
+                records: [{
+                  id: '25d81cbe-9686-11ea-bb37-0242ac130002',
+                  name: 'AP Holdings 2',
+                  recordTypes: ['HOLDINGS'],
+                  transformations: [],
+                  userInfo: {
+                    firstName: 'Donald',
+                    lastName: 'S',
+                    userName: 'system_process',
+                  },
+                  outputFormat: 'MARC',
+                  metadata: {
+                    createdDate: '2018-12-04T01:29:36.531+0000',
+                    updatedDate: '2018-12-04T01:29:36.531+0000',
+                  },
+                }],
+              }}
+              onCancel={noop}
+            />
+          </Router>
+        </Paneset>,
+        translationsProperties,
+      );
+    });
+
+    it('should display no value in description', () => {
+      expect(mappingProfileDetails.summary.description.value.text).to.equal('-');
+    });
+
+    it('should not display transformation list', () => {
+      expect(mappingProfileDetails.transformations.list.isPresent).to.be.false;
+    });
+  });
+
+  describe('rendering mapping profile details', function () {
+    const handleSubmitSpy = sinon.spy();
+
+    beforeEach(async function () {
+      handleSubmitSpy.resetHistory();
```

Comment:
I am not sure `handleSubmitSpy` is needed here.

Reply:
Sure, you're right, it is extra work, thanks for catching it